builtin/mainmenu/tab_about.lua

```diff
@@ -43,18 +43,19 @@ local core_team = {
 -- For updating active/previous contributors, see the script in ./util/gather_git_credits.py
 
 local active_contributors = {
-	"Wuzzy [Features, translations, devtest]",
-	"Lars Müller [Bugfixes and entity features]",
-	"paradust7 [Bugfixes]",
-	"ROllerozxa [Bugfixes, Android]",
-	"srifqi [Android, translations]",
-	"Lexi Hale [Particlespawner animation]",
+	"Wuzzy [Features, translations, documentation]",
+	"numzero [Optimizations, work on OpenGL driver]",
+	"ROllerozxa [Bugfixes, Mainmenu]",
+	"Lars Müller [Bugfixes]",
+	"AFCMS [Documentation]",
 	"savilli [Bugfixes]",
 	"fluxionary [Bugfixes]",
-	"Gregor Parzefall [Bugfixes]",
+	"Thresher [Documentation]",
```

Comment:
Please update my name.

Reply:
`Bradley Pierce` or `Bradley Pierce (Thresher)` or `Bradley Pierce (Bituvo)`?

Reply:
`Bradley Pierce (Thresher)` please. I chose the name Bituvo a while ago and I've grown old of it...